app/src/test/java/com/simpletrack/TaskTests.kt

```diff
@@ -5,6 +5,7 @@ import org.junit.Assert.assertEquals
 import java.lang.Thread.sleep
 
 import Task
+import java.util.*
```

Comment:
/github/workspace/app/src/test/java/com/simpletrack/TaskTests.kt:8:1: Wildcard import (cannot be auto-corrected)